systemcontractindex/stakingindex/voteview.go

```diff
@@ -113,10 +145,9 @@ func (s *voteView) CandidateStakeVotes(ctx context.Context, candidate address.Ad
 	return s.cur.Votes(featureCtx, candidate.String())
 }
 
-func (s *voteView) CreatePreStates(ctx context.Context, br BucketReader) error {
+func (s *voteView) CreatePreStates(ctx context.Context) error {
 	blkCtx := protocol.MustGetBlockCtx(ctx)
 	s.height = blkCtx.BlockHeight
-	s.store = newBucketStore(br)
```

Comment:
should reset the store in `CreatePreStates`,  otherwise it keeps accumulated dirties but with latest indexer